src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlAttributes.cs

```diff
@@ -91,6 +93,7 @@ private static Type IgnoreAttribute
         /// <devdoc>
         ///    <para>[To be supplied.]</para>
         /// </devdoc>
+        [RequiresUnreferencedCode(XmlSerializer.LinkerSerializationMessage)]
```

Comment:
I'm not sure I see what requires unreferenced code in this constructor.

Reply:
It calls the `IgnoreAttribute` property (which is RUC above)